Microsoft.Packaging.Tools/tasks/Trimming/FileNode.cs

```diff
@@ -67,27 +92,44 @@ public void PopulateDependencies(Dictionary<string, FileNode> allFiles, bool pre
                     if (peReader.HasMetadata)
                     {
                         var reader = peReader.GetMetadataReader();
+
+                        var includeDependencies = true;
+
+                        // map of facade handles to enable quickly getting to FileNode without repeatedly looking up by name
+                        var facadeHandles = new Dictionary<AssemblyReferenceHandle, FileNode>();
+
+                        if (IsFullFacade(reader))
```

Comment:
Isn't this also potentially interesting for partial facades? I could imagine for example some assemblies which have on a couple types defined and forwards that are never followed.